Makefile

```diff
@@ -0,0 +1,5 @@
+build-corefn:
```

Comment:
```
.PHONY: build-corefn
build-corefn: ...
```

Even better: just `corefn` instead of `build-corefn` since you're already `make`ing it.

Sample applies for `regen-scheme`.

Also the file is missing a newline at the bottom.